components/OutivityDetail/index.js

```diff
@@ -22,29 +24,27 @@ export default function OutivityDetail({ outivity, onDeleteOutivity }) {
       </Head>
       <main>
         <article>
-          <ul>
-            <li>
-              <h2>{outivity.title}</h2>
-              <StyledOutivityImage
-                src={outivity.image}
-                alt={outivity.title}
-                width={300}
-                height={200}
-                layout="fixed"
-              />
-              <p>
-                <strong>Location: </strong> {outivity.area},
-                <StyledCountryName>{outivity.country}</StyledCountryName>
-              </p>
+          <h2>{outivity.title}</h2>
```

Comment:
To make the headline on the details page more consistent with the headline in the edit form, we suggest, you also make this one a `<h1>` headline. It's also the first headline of the page, so it'd make more sense anyway 😉
```suggestion
          <h1>{outivity.title}</h1>
```

Reply:
You are sharp-eyed  👀  – thanks! 
We changed it to a `<h1>`